crates/wasi-common/src/error.rs

```diff
@@ -97,38 +97,23 @@ impl WasiError {
 impl fmt::Display for WasiError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match *self {
-            _ => write!(f, "{:?}", self),
+            _ => write!(f, "{}", self),
```

Comment:
Can this whole `fmt::Display` impl be removed entirely now?

Reply:
We can get rid of it, but we still need `WasiError` to derive `fmt::Display` which however can be done more efficiently using `#[error(..)]` attribute. See 3b392db.